### PR TITLE
Features new pages

### PR DESCRIPTION
**Pull request by redmarverick**
- Created page works
- Created page About me
- Done styles both desktop and mobile

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -170,4 +396,199 @@ a:link {
...
+
+  #w-title {
+    font-size: 10vw;
+    padding: 5vw;
+    margin-bottom: 2vw;
+  }
+
+  #works {
+    padding: 5vw;
+  }
```
Comment (tsohleDev) [inline styles.css]: - [x] I kindly suggest that you make the works section a ``grid`` layout as it was a requirement of the project ☺️ 

> In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card.

File: styles.css [hunk 2]
```diff
@@ -170,4 +396,199 @@ a:link {
...
+
+  .separator {
+    width: 7vw;
+    border-radius: 10px;
+    height: 0.8vw;
+    display: flex;
+    background-color: #d92424;
```
Comment (tsohleDev) [inline styles.css]: - [ ] on the design the color swatch included  #36B37E  you replaced it with  #d0d9d4, I suggest you stick to the Figma design, as it is important for a front end to create a design exactly as it is without making personal changes


| Your design | bad color 1 | bad color 2 | bad color 3 |
| --- | --- | --- | --- |
| Figma's design | good color1 | good color2 | good color3 |

File: index.html
Comment (tsohleDev) [inline index.html:1]: [COMPLIMENT] Very organised and concise files, keep it up 👍 🏆

**Review by tsohleDev — CHANGES_REQUESTED** [review]
## Hi Micronaut @redmarverick 

Good job so far!
There are some issues that you still need to work on.

163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856

## Highlights!
- README template ✔️ 
- Descriptive commits ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- All requirements met ❌

Hello ✋, you have done well creating your project 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

## Compliment
_Every comment with the [COMPLIMENT] prefix is not  a request, just a compliment at your skills_

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@tsohleDev` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (redmarverick) [reply styles.css]: grid applied to works section

File: styles.css (hunk 2 above)
Comment (redmarverick) [reply styles.css]: all color pallete altered under request

**Comment by redmarverick** [comment]
@tsohleDev cant open a new review request as its showing as open on dashboard

**Review by Lookingfor00 — APPROVED** [review]
# Approved 🟢
Hi @redmarverick, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
allfather-approves-approved

## Highlights
- All linters pass 🍏 
- Professional documented project 📖
- Clean and well organized code 🧼 😄  

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@Trast00) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**